Corona-Warn-App/build.gradle

```diff
@@ -173,7 +173,8 @@ android {
                     "-Xno-kotlin-nothing-value-exception",
                     "-Xuse-experimental=kotlinx.coroutines.ExperimentalCoroutinesApi",
                     "-Xuse-experimental=kotlinx.coroutines.FlowPreview",
-                    "-Xuse-experimental=kotlin.time.ExperimentalTime"
+                    "-Xuse-experimental=kotlin.time.ExperimentalTime",
+                    "-Xopt-in=kotlin.RequiresOptIn"
```

Comment:
Did this PR add something that uses it? Can't spot it :thinking:

Reply:
Not in here , it was part of Swipe feature, I used `OptIn` annotation which is still in preview